server/src/main/java/com.github.jsappz.lamegame/server/database/actions/UserActions.java

```diff
@@ -0,0 +1,37 @@
+package com.github.jsappz.lamegame.server.database.actions;
+
+import com.github.jsappz.lamegame.server.database.DatabaseAction;
+import com.github.jsappz.lamegame.server.model.User;
+
+import java.sql.*;
```

Comment:
how do we feel about star imports? 
do we want to never do them? have a limit? 25? 10?

Reply:
Thats up to you. Intellij just auto did these. I think it makes compilation a little faster